src/utils/narrow.js

```diff
@@ -10,7 +10,7 @@ export const homeNarrow: Narrow = [];
 
 export const homeNarrowStr: string = '[]';
 
-export const isHomeNarrow = (narrow: Narrow): boolean => narrow.length === 0;
+export const isHomeNarrow = (narrow: Narrow): boolean => (narrow ? narrow.length === 0 : false);
```

Comment:
This is still not robust enough. What if `narrow` is a string of `'Hello, world'`?
Do this `Array.isArray(narrow) && narrow.length === 0`.